Formula/gauge.rb

```diff
@@ -17,25 +15,16 @@ class Gauge < Formula
   depends_on "go" => :build
   depends_on "godep" => :build
 
-  go_resource "github.com/getgauge/gauge_screenshot" do
-    url "https://github.com/getgauge/gauge_screenshot.git",
-        :revision => "f16bcf089e263db525a255c9f9621ca0270ff1d3"
-  end
-
   def install
     ENV["GOPATH"] = buildpath
     ENV["GOROOT"] = Formula["go"].opt_libexec
 
     # Avoid executing `go get`
-    inreplace "build/make.go", /\tgetGaugeScreenshot\(\)\n/, ""
+    inreplace "build/make.go", "", ""
```

Comment:
Why are we replacing the empty string with the empty string?

Reply:
You don't have clearance for that information, obviously